warehouse/clickhouse/clickhouse.go

```diff
@@ -123,24 +115,37 @@ var clickhouseDataTypesMapToRudder = map[string]string{
 	"SimpleAggregateFunction(anyLast, Nullable(Int16))":    "int",
 	"SimpleAggregateFunction(anyLast, Nullable(Int32))":    "int",
 	"SimpleAggregateFunction(anyLast, Nullable(Int64))":    "int",
-	"SimpleAggregateFunction(anyLast, Nullable(Floats32))": "float",
-	"SimpleAggregateFunction(anyLast, Nullable(Floats64))": "float",
+	"SimpleAggregateFunction(anyLast, Nullable(Float32))":  "float",
```

Comment:
This seems to be a bug caught during writing Unit tests. Because of this, during the fetching of the schema, these are part of the unrecognized schema.
Data has went to rudder discards instead of their corresponding tables.